tools-local/unix/search-tool.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+# Searches for the specified tool in the environment path, and the path within the repository as specified in the .toolversions file.
+
+usage()
+{
+    echo "Usage: $0 ToolName [-StrictToolVersionMatch]"
+    echo "  ToolName: Name of the tool to download."
+    echo "  (Optional) -StrictToolVersionMatch: If specified then, search will ensure that the version of the tool searched is the declared version."
+    echo "                                      Otherwise, search will attempt to find a version of the tool, which may not be the declared version."
+    echo ""
+    echo "Searches for the tool in the environment path, and the path within the repository as specified in the .toolversions file."
```

Comment:
Good point. Updated as "Searches for the tool in the environment path, and a path specified for the tool in the .toolversions file."